internal/controller/proxmoxcluster_controller.go

```diff
@@ -265,8 +265,8 @@ func (r *ProxmoxClusterReconciler) reconcileFailedClusterState(clusterScope *sco
 		if err != nil {
 			return errors.Wrap(err, "failed to init patch helper")
 		}
-		clusterScope.Cluster.Status.FailureMessage = nil
-		clusterScope.Cluster.Status.FailureReason = nil
+		// clusterScope.Cluster.Status.FailureMessage = nil
```

Comment:
why comment those?

Reply:
I meant to remove them. Cluster.Status no longer has these fields.

Reply:
Impossible, those fields are required for the contract spec.